src/components/Home/EventModal/index.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
+import ModalHeader from './modal-header';
+import ModalBody from './modal-body';
 
-const EventModal = (props) => {
+const EventModal = ({townhall}) => {
+  
   return (
-    <div className="modal fade event-modal" tabIndex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
-      <div className="modal-dialog modal-lg" role="document">
-        <div className="modal-content">
+    <React.Fragment>
+    {
+      townhall 
+        ? <div className="modal fade event-modal" tabIndex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
```

Comment:
for this type of check I like `townhall && <></>`  instead of `townhall ? <></> : null`
```suggestion
        && <div className="modal fade event-modal" tabIndex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
```